packages/block-library/src/calendar/style.scss

```diff
@@ -2,31 +2,22 @@
 	text-align: center;
 
 	th,
-	tbody td {
+	td {
 		padding: 0.25em;
-		border: 1px solid $gray-300;
-	}
-
-	tfoot td {
-		border: none;
+		border: 1px solid;
 	}
 
 	table {
 		width: 100%;
 		border-collapse: collapse;
-	}
-
-	table th {
-		font-weight: 400;
-		background: $gray-300;
-	}
 
-	a {
-		text-decoration: underline;
+		// Keep the hard-coded header background color for backward compatibility.
+		&:not(.has-text-color):not(.has-background) th {
+			background: $gray-300;
+		}
 	}
 
-	table tbody,
-	table caption {
-		color: #40464d;
+	thead {
+		border-bottom: 3px solid;
```

Comment:
Sorry it looks like my comment didn't send the first time:

What's the reason for adding this? I think we should be leaning towards removing styles rather than adding them

Reply:
I would like you to watch this video regarding the style comparison with table blocks in my previous comment, The table block has a similar border under `thead`.
I have made the same changes to the calendar block for style consistency, but do you think this is unnecessary?

Reply:
+1 to not introducing any new styles.

Reply:
Ok, I'm going to remove this style 👍

Reply:
Fixed by 8b4e335